2-mandatory/8-journey-planner.js

```diff
@@ -19,10 +19,16 @@
   What's better about each approach?
 */
 
-function journeyPlanner(locations, transportMode) {}
+const journeyPlanner = (locations, transportMode) => {
+  arr = [];
```

Comment:
Ohhh, you have written `arr` here, but not used `const` or `let` to declare it!!! Super weird this test passes, as this is not correct. But don't worry! We all make this mistake a million times! :)